test/metabase/lib/drill_thru_test.cljc

```diff
@@ -107,6 +107,7 @@
 (deftest ^:parallel table-view-available-drill-thrus-headers-pk-test
   (testing "column headers: click on"
     (testing "primary key - column filter (default: Is), sort, summarize (distinct only)"
+      (lib.drill-thru.tu/coverage-available! 0 0 :header :pk)
```

Comment:
Would this be better if we just made a wrapper around `available-drill-thrus` that looked at the query shape and context shape and recorded this stuff, and then made sure  `lib.drill-thru.tu/test-available-drill-thrus` and other stuff in the drill thru test namespaces used it? Same for `lib/drill-thru`.